build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 subprojects {
 
     group = 'edu.wisc.my.restproxy'
-    version = '2.1.3-SNAPSHOT'
+    version = '2.1.3'
```

Comment:
@apetro I believe this revision is stable -- I made sure the unit tests still pass, and verified that STAR can still build and run against it.